app/services/weather_man.rb

```diff
@@ -1,32 +1,40 @@
 class WeatherMan
-  @w_api = Wunderground.new(ENV["WUNDERGROUND_KEY"])
+  @api = Wunderground.new(ENV["WUNDERGROUND_KEY"])
 
-  def self.key_for(zip_code, time)
+  def self.api
+    @api
+  end
+
+  def self.key_for(location, time)
     format = '%Y-%m-%d'
     format += 'H%H' if time.to_date === Date.today
 
-    "outdoor_temp_for_#{zip_code}_on_#{time.strftime(format)}"
+    "outdoor_temp_for_#{location}_on_#{time.strftime(format)}"
   end
 
-  def self.current_outdoor_temp(zip_code, throttle = 9)
-    return nil if !zip_code
-    Rails.cache.fetch(key_for(zip_code, Time.zone.now), :expires_in => 1.hour) do
+  def self.current_outdoor_temp(location, throttle = 9)
+    return nil if !location
+    Rails.cache.fetch(key_for(location, Time.zone.now), :expires_in => 1.hour) do
       sleep throttle
-      json_object = @w_api.conditions_for(zip_code)
+      json_object = @api.conditions_for(location)
       json_object["current_observation"]["temp_f"]
     end
   end
 
-  def self.outdoor_temp_for(time:, zip_code:, throttle: 9)
-    wunderground_history = fetch_wunderground_history(time, zip_code, throttle)
+  # This method will need to be restored once we are in other cities or support
+  # hobbyists. This is a stopgap because it makes our outdoor temperatures
+  # automatically admissible as evidence and we currently are only in one city.
+  def self.outdoor_temp_for(time:, location:, throttle: 9)
+    location = 'knyc' # temporarily force all readings to use NOAA central park station
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.